lib/phoenix_live_view/html_engine.ex

```diff
@@ -762,4 +765,33 @@ defmodule Phoenix.LiveView.HTMLEngine do
   defp void?(_), do: false
 
   defp to_location(%{line: line, column: column}), do: [line: line, column: column]
+
+  @attrs_with_required_ids ["phx-update", "phx-hook"]
```

Comment:
> When using phx-update, a unique DOM ID must always be set in the container. If using "append" or "prepend", a DOM ID must also be set for each child. 

Docs also says about childs when using `append | prepend`. Not sure how to validate though since where we don't have the tree 🤔

Reply:
BTW, other checks we could do:

1. check if `phx-update=` contains `append | prepend | ignore`
1. check if `phx-hook=` contains a string value
1. check if `phx-value` and friends have string values.

Make sense and any other validations we want to introduce?

Reply:
BTW, other checks we could do:

1. check if `phx-update=` contains `append | prepend | ignore`
1. check if `phx-hook=` contains a string value
1. check if `phx-value` and friends have string values.

Make sense and any other validations we want to introduce?

Reply:
I wouldnt check for string values because everything kinda becomes a string at the end but checking for phx-update value would be cool.